src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/changes/managingofficer/officer/Officer.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.changes.managingofficer.officer;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels.Address;
+
+import java.time.LocalDate;
+
+public class Officer {
+    @JsonProperty("actionDate")
+    private LocalDate actionDate;
+    @JsonProperty("serviceAddress")
+    private Address serviceAddress;
+    @JsonProperty("startDate")
```

Comment:
startDate will be defined so can't be null I think but can be null in registration.